drivers/led/issi/is31fl3741.c

```diff
@@ -104,11 +104,11 @@ void IS31FL3741_write_register(uint8_t addr, uint8_t reg, uint8_t data) {
 }
 
 bool IS31FL3741_write_pwm_buffer(uint8_t addr, uint8_t *pwm_buffer) {
-    // Assume PG1 is already selected
+    // Assume PG0 is already selected
 
-    for (int i = 0; i < 342; i += 18) {
+    for (int i = 0; i < ISSI_MAX_LEDS; i += 18) {
```

Comment:
This change actually introduces a bug — it causes a buffer overrun, because `ISSI_MAX_LEDS` = 351 is not an integer multiple of 18, and after the change the last iteration of this loop will run with `i` = 342 and try to access the LED data for indices 342…359, going past the end of the array.  There is even some code after the loop which handles the leftover 9 LEDs, but this loop needs to end one iteration earlier.

If you really want to remove that magic 342 from the code, you would need to do something like this:
```suggestion
    for (int i = 0; i < (ISSI_MAX_LEDS / 18) * 18; i += 18) {
```
(But then there are more magic numbers in the code which deals with the leftover 9 LEDs.)

The only way to have just a single loop is to set the chunk length to GCD(180, 351) = 9, but that would double the protocol overhead, which is probably not desired.  Another possible solution is to have two separate loops for two pages, handling PG0 (180 LEDs) with the chunk length of 18 (as it is now), and then handling PG1 (171 LEDs) with the chunk length of 19 (171 = 3*3*19), but that looks even more complicated and specific to the particular number of LEDs than the current code.

Reply:
I agree with @zvecr. I first replaced it because the magic number looked somewhat arbitrary. I think it would make most sense if we introduce a variable for `buffer_size` and update all magic numbers accordingly. But since this would be slightly out of scope of the initial PR, it may make sense to keep that for a another PR. Let me know what you think.